nonce_test.go

```diff
@@ -4,15 +4,55 @@
 package eat
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestNonce_NewNonce_ok(t *testing.T) {
+	for i := MinNonceSize; i <= MaxNonceSize; i++ {
+		tv := make([]byte, i)
+
+		nonce, err := NewNonce(tv)
+
```

Comment:
Is this an extra line added intentionallY?

Reply:
which extra line?

Reply:
Line 16

Reply:
which extra line?

Reply:
yes